### DISCUSSION TITLE
Kegland RAPT...

### DISCUSSION THREAD

**Discussion by gingerneil** [universam1/iSpindel#574] [General]
I see kegland has launched the RAPT. It looks REMARKABLY similar to the ispindel! Its still twice the price of buying a ready built device here in the UK on somewhere like ebay. I've not dug into it in much detail, but I don't think it supports many (any?) external data services. Still... At least it would appear to validate the hard work put into ispindel over the years, now that someone other than Tilt are commercialising the concept into a full product.
I wonder if this will have any impact on the ispindel development, or community?

**Comment by ErikdBr** [comment]
Their not the only ones: https://brewbrain.nl/product/float/#retailers and I don't think it has any effect on the iSpindel development or community.
